src/b2c-ms-graph.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <RootNamespace>b2c_ms_graph</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Graph.Auth" Version="1.0.0-preview.2" />
+    <PackageReference Include="Microsoft.Graph.Beta" Version="0.12.0-preview" />
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.5.1" />
```

Comment:
>4.5.1 [](start = 67, length = 5)

we have 4.8.2 out now...maybe update at some point? What is a good way to keep you informed of new packages? We release every 3-4 weeks.

Reply:
@jennyf19 Updated all packages to latest - good call!

Reply:
Oh, and on the update notification - maybe an email to the GTP alias?

@yoelhor how would your team like to be notified when new MSAL libs are released so you can update your samples?